### ISSUE TITLE
Support a consistent interpretation of WCAG across government

### ISSUE THREAD

**Issue by davidc-gds** [alphagov/govuk-design-system#2451]
## What

**Timeline:** Estimated from January 2023 to October 2023
**Priority level:** 5 – what makes it easier for service teams to do their jobs

### Potential tasks

- Establish a link with the monitoring body to make sure GOV.UK Frontend meets WCAG 2.2, based on their testing criteria
- Align with and support the [WCAG testing guide](https://github.com/alphagov/wcag-primer/wiki)
- Update the WCAG primer with any new insights we learn from updating GOV.UK Frontend to WCAG 2.2
- Explore opportunities to better share WCAG and WCAG testing knowledge with contributors

## Epic lead
Not yet assigned

## Why
The GOV.UK Design System is a major layer of frontend infrastructure for many UK government services and websites. The Accessibility Monitoring team assesses services and websites for WCAG compliance, based on a particular interpretation of WCAG. We should therefore align our WCAG interpretations, so that the base layer of GOV.UK Frontend doesn't cause repeated blockers when it comes to accessibility monitoring.

## Who needs to work on this
Internal to team: accessibility specialist, product manager

External to team: Accessibility Monitoring team, other accessibility specialists

## Who needs to review this
Unknown

## Done when

- [ ] Thing to do

**Comment by mgifford** [comment]
Please also include the great work you are doing in blogging about this work. Your communications is really quite inflential!

**Comment by davidc-gds** [comment]
A small update on the work we've done so far on this lower-priority accessibility activity (prompted by @mgifford 's note):

- Provided input to the Accessibility Monitoring Body on how to add mobile testing guidance to the [accessibility testing guide](https://github.com/alphagov/wcag-primer/wiki)
- Provided input on an upcoming WCAG 2.2 refresh of the content for the CDDO Guidance page, "[Doing a basic accessibility check if you cannot do a detailed one](https://www.gov.uk/government/publications/doing-a-basic-accessibility-check-if-you-cant-do-a-detailed-one/doing-a-basic-accessibility-check-if-you-cant-do-a-detailed-one)"
